gateway-js/src/index.ts

```diff
@@ -230,17 +231,22 @@ export class ApolloGateway implements GraphQLService {
 
     this.experimental_pollInterval = config?.experimental_pollInterval;
 
-    // Do not use this unless advised by Apollo staff to do so
-    // 1. If config is explicitly set to a `string` or `null` in code, use it
-    // 2. Else, if the env var is set, use that
-    // 3. Else, default to `null`
-    this.experimental_schemaConfigDeliveryEndpoint = isPrecomposedManagedConfig(
-      this.config,
-    )
-      ? this.config.experimental_schemaConfigDeliveryEndpoint
-      : this.config.experimental_schemaConfigDeliveryEndpoint === null
-      ? null
-      : process.env.APOLLO_SCHEMA_CONFIG_DELIVERY_ENDPOINT ?? null;
+    // 1. If config is set to a `string`, use it
```

Comment:
I think this should be reordered as
  string
  null
  env var
  otherwise default

Reply:
9f9a2b06b8bec3e5f2cbe2038c96036c5b0ed22b

Reply:
Updated PR description as well